conjure-undertow-lib/src/main/java/com/palantir/conjure/java/undertow/lib/RequestContext.java

```diff
@@ -44,4 +45,10 @@
      * Returns all query parameters associated with the current request.
      */
     ListMultimap<String, String> queryParameters();
+
+    /**
+     * Associates an {@link Arg} with the current request for observability.
+     * Implementations may choose to include this data in the request log.
```

Comment:
"Implementation may choose to use this data for request logging and other observability needs"?